README.md

```diff
@@ -1,5 +1,5 @@
 <p align="center">
-  <img height="100" src="https://phylum.io/logo/dark-bckg.svg">
+  <img height="100" src="https://raw.githubusercontent.com/phylum-dev/cli/main/assets/dark-bckg.svg">
```

Comment:
Does this work?
```suggestion
  <img height="100" src="assets/dark-bckg.svg">
```

Reply:
this will work, but keeping it in sync with `quickstart.md`